hardly/handlers/distgit.py

```diff
@@ -31,14 +31,46 @@ def __init__(
             job_config=job_config,
             event=event,
         )
+        self.mr_identifier = event.get("identifier")
+        self.mr_title = event.get("title")
+        self.mr_description = event.get("description")
+        # Gitlab specific, but hardly is too
+        self.mr_url = (
+            f"{event.get('project_url')}/-/merge_requests/{self.mr_identifier}"
+        )
+        self.source_project_url = event.get("source_project_url")
+        self.target_repo_branch = event.get("target_repo_branch")
 
     def run(self) -> TaskResults:
         """
         If user creates a merge-request on the source-git repository,
         create a matching merge-request to the dist-git repository.
         """
+        if self.target_repo_branch != "c9s":
+            return TaskResults(success=True, details={})
 
-        logger.debug(
-            "RUNNING DistGitPRHandler, Implement me !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+        source_project = self.service_config.get_project(url=self.source_project_url)
+        self.local_project = LocalProject(
+            git_project=source_project,
+            ref=self.data.commit_sha,
+            working_dir=self.service_config.command_handler_work_dir,
         )
-        return TaskResults(success=True, details={})
+
+        self.api = PackitAPI(
+            config=self.service_config,
+            package_config=self.package_config,
+            upstream_local_project=self.local_project,
+        )
+
+        dg_mr = self.api.sync_release(
```

Comment:
We should use `update_dist_git` here.

Reply:
We are, via `sync_release()`.
I initially also thought I'd use `update_dist_git` directly, but then I realized I'd basically be a copy of `sync_release()`.